### DISCUSSION TITLE
What date the release of the new version PragmataPro ?

### DISCUSSION THREAD

**Discussion by LappA17** [fabrizioschiavi/pragmatapro#258] [General]
Hello :) I really like your font PragmataPro and I'm interested in what date the release of the new version 0.830 is being prepared for or maybe the name of the version will be different ? Please share information with us, thank you !

**Comment by fabrizioschiavi** [comment]
Thanks for your interest in PragmataPro!
I started working at the update months ago but I can't work full time because of some new custom works.
Don't worry, I can't forget PragmataPro 0.830 project. The new release has already some **big** improvements.
Keep in touch
